### PR TITLE
Add fetchOHLCV to bithumb

### PR DESCRIPTION
**Pull request by ne2030**
When i tests basic-chart in examples, bithumb doesn't work well.
So i checked bithumb api and add candlestick request and fetchOHLCV method to bithumb.

### CODE REVIEW

**Comment by kroitor** [comment]
Hi! First of all, thanks for the PR! We really appreciate all contributions! One question, though: can you please align it with the other exchanges in terms of syntax?

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

**Comment by kroitor** [comment]
We need to use `this.parseOHLCVs` or loops instead of `.map()` with closures.

**Comment by ne2030** [comment]
Oh, i see. I'll change that part.
Thanks!

File: js/bithumb.js [hunk 1]
```diff
@@ -349,6 +350,39 @@ module.exports = class bithumb extends Exchange {
...
+        //         "0.02501786",
+        //         "0.02501786",
+        //         "0.02501786",
+        //         "0.02501786",
+        //         "0.0000",
+        //         1591488000000
+        //     ]
+        //
+        return [
+            this.safeInteger (ohlcv, 0),
```
Comment (kroitor) [inline js/bithumb.js:365]: I think the timestamp should be `this.safeInteger (ohlcv, 5)` here, lmk if we're missing something.

Comment (ne2030) [reply js/bithumb.js:365]: Oh, i think i didn't changed comment when i copy paste from other exchange;
Code is right, just comment is wrong sorry.

**Pull request merged**